src/main/java/com/machine/learning/Main.java

```diff
@@ -49,6 +50,7 @@ public static void main(String ... args) {
 	}
         classifierRegistry.put("naive-bayes", new NaiveBayes());
         classifierRegistry.put("tree-naive-bayes", new TreeAugmentedNaiveBayes());
+        classifierRegistry.put("iterative-dichotomiser-3", new ID3());
```

Comment:
Call this id3 please. I don't want to type in a bunch of stuff to only run your algorithm